web_test.go

```diff
@@ -156,8 +147,6 @@ func TestForwardHandlerHandleStatusEvent(t *testing.T) {
 
 	// bad status
 	tempBody := tempTaskBody("TASK_BATMAN")
-	recorder := httptest.NewRecorder()
-	handler.HandleStatusEvent(recorder, tempBody)
-	assert.Equal(t, 500, recorder.Code)
-	assert.Equal(t, "unknown task status\n", recorder.Body.String())
+	err := handler.HandleStatusEvent(tempBody)
+	assert.NotNil(t, err)
```

Comment:
please change this assertion so that it's asserting something meaningfully equal.